flake.nix

```diff
@@ -245,10 +248,12 @@
           mkdir -p $out/bin
           for srcBin in $src/bin/*; do
             outBin="$out/bin/$(basename $srcBin)"
-            # [dirty] must have 7 characters to match dummyRev (hard coded in
-            # nickel-lang-cli and nickel-lang-lsp)
-            # we have to pad them out to the same length as dummyRev so they fit
-            # in the same spot in the binary
+        '' +
+        # [dirty] must have 7 characters to match dummyRev (hard coded in
+        # nickel-lang-cli and nickel-lang-lsp)
+        # we have to pad them out to the same length as dummyRev so they fit
+        # in the same spot in the binary
```

Comment:
This avoid having comments be part of the output, and thus causing a rebuild if we ever change them.